docs/developer.md

```diff
@@ -380,8 +380,12 @@ And to see a list of available templates that can be resolved for Typst, you can
 
 https://api.mystmd.org/templates/typst
 
-
 ## How to make a release
+
+:::{important} Informing other developers
+Making a release of MyST components may require other stakeholders to update their deployments of MyST. In order to minimise disruption, contributors should inform other developers of their intention to make a release in the [`#development` Discord channel](https://discord.com/channels/1083088970059096114/1177359316886503464). This step may be skipped if the benefits of making an unannounced release outweigh the risks, such as mitigating a security vulnerability or preventing data loss.
```

Comment:
I think this can't be skipped on the informing side. "Making a release now." Is not hard to inform! If this should not be public, then coordinate in private channels.

It would be good to coordinate: "planning to make a release today to .., any one have anything else they are planning to get in planning this by the end of my day in a few hours?"

Reply:
It feels like an easy and safe to try start would be a team policy like:

- Before making a release for any repository, make a post to `#development` stating your intent to do so.
- Use your best judgment about whether to wait any amount of time before taking action.

And then if we run into friction because we released too quickly, then we can add stronger guidelines for waiting for feedback. Or if it's too noisy in `#development` and we miss these messages, we can discuss adding a new channel.

Reply:
Made a `release_coordination` channel.